installer/tue-install-impl.bash

```diff
@@ -1048,6 +1048,55 @@ function tue-install-snap-now
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+function tue-install-gem
+{
+    tue-install-debug "tue-install-gem $*"
+
+    if [ -z "$1" ]
+    then
+        tue-install-error "Invalid tue-install-gem call: needs package as argument."
+    fi
+    tue-install-debug "Adding $1 to gem list"
+    TUE_INSTALL_GEMS="$1 $TUE_INSTALL_GEMS"
+}
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+function tue-install-gem-now
+{
+    tue-install-debug "tue-install-gem-now $*"
+
+    if [ -z "$1" ]
+    then
+        tue-install-error "Invalid tue-install-gem-now call: needs package as argument."
+    fi
+
+    tue-install-system-now rubygems-integration
+
+    local gems_to_install gems_installed
+    gems_to_install=""
+    gems_installed=$(gem list)
+    # shellcheck disable=SC2048
+    for pkg in $*
+    do
+        if [[ ! $gems_installed == *$pkg* ]] # Check if pkg is not already installed
+        then
+            gems_to_install="$gems_to_install $pkg"
+            tue-install-debug "gem pkg: $pkg is not yet installed"
+        else
+            tue-install-debug "gems pkg: $pkg is already installed"
+        fi
+    done
+
+    if [ -n "$gems_to_install" ]
+    then
+        # shellcheck disable=SC2086
+        tue-install-pipe sudo gem install $gems_to_install || tue-install-error "An error occurred while installing gem packages." #<<< yes
```

Comment:
Does `gem` always require `sudo`?

Reply:
That was how it was used in the gitsu target. And running `gem install gitsu` does indeed return a permission error.